src/index.ts

```diff
@@ -14,7 +14,7 @@ import { CardApi } from './card'
 import { CardProductApi } from './card-product'
 import { WebhooksApi } from './webhooks'
 import { FundingGatewayApi } from './funding-gateway'
-
+import { KycApi } from './kyc'
```

Comment:
The convention is to have the blank line after the `import` and before anything else.